README.md

```diff
@@ -94,6 +94,10 @@ module.exports = ({ env }) => ({
 | url | The trigger URL for the website build. | String | Yes |
 | headers | Any headers to send along with the request. | Object | No |
 | body | Any body data to send along with the request. | Object | No |
+| secret | Secret key to sign body of the request with. | Object | No |
+| secret.hash | Algorith to use for signing e.g. sha1, sha256, etc. | string | Yes |
+| secret.headerKey | Header key where the signature should be stored. | string | Yes |
+| secret.secretKey | Secret key to sign body with. | string | Yes |
```

Comment:
I believe we should use the following key names over the current to avoid confusion between common terms.

```suggestion
| signature | The signature to send as a header with the request | object | No |
| signature.alogorithm | the hash algorithm to use for signing e.g. `sha1`, `sha256`, etc. | string | Yes |
| signature.headerName | The name of the header to add the signature to. | string | Yes |
| signature.secret | The secret key to sign the request body with. | string | Yes |
```